daemon.go

```diff
@@ -146,3 +147,41 @@ func (d *Daemon) listen() {
 		go d.handleConn(c)
 	}
 }
+
+func clearUnixSockets(maddr ma.Multiaddr) error {
```

Comment:
Can we call this var `path`? it conveys what we expect it to be.

Reply:
Yes, done